.storybook/config.js

```diff
@@ -1,7 +1,17 @@
 import { configure } from '@storybook/react';
-import './provider';
+import './global.css';
+import { addDecorator } from '@storybook/react';
+import React from 'react';
+import BandwidthProvider from 'components/BandwidthProvider';
+import { Router } from 'react-router';
+import { createMemoryHistory } from 'history';
+
+const history = createMemoryHistory('/');
 
 function loadStories() {
+  // pull in provider, see https://github.com/storybooks/storybook/issues/1844
+  addDecorator(getStory => <BandwidthProvider>{getStory()}</BandwidthProvider>);
```

Comment:
I hope this really fixes this problem, because I tried similar solutions a bunch and came up empty-handed.